Braintree/Payments/@Public/BTPaymentProvider.h

```diff
@@ -18,6 +18,29 @@ typedef NS_ENUM(NSInteger, BTPaymentProviderType) {
     BTPaymentProviderTypeApplePay
 };
 
+/// Status of the last (or ongoing) transaction
```

Comment:
> transaction

We should probably favor the term "payment method creation" to avoid confusion with Braintree's typical notion of [_transaction_ creation](https://developers.braintreepayments.com/ios+ruby/sdk/server/transaction-processing/create#minimal-example).